openedx/core/djangoapps/programs/utils.py

```diff
@@ -62,11 +65,26 @@ def get_program_and_course_data(site, user, program_uuid, mobile_only=False):
     return program_data, course_data
 
 
+def get_buy_subscription_url(*skus, program_uuid=None):
+    """
+    Construct the url for the subscription purchase page.
+    """
+    query_params = {'sku': skus}
+    print(f"Query Params are: {query_params}")
```

Comment:
Can we delete this? Was this used for testing? Or is this something that should be properly logged?